### PR TITLE
feat: hyperlink contact info into Teams.md

### LINKED ISSUE #15: Hyperlink contact information in the `Team.md` file
## Steps

1. Go to [the `Americas_01` section of this spreadsheet](https://docs.google.com/spreadsheets/d/1L7iLTDgX0UJrEk97RKk7RIdniPOoXDQU/edit?usp=sharing&ouid=106657036308661264833&rtpof=true&sd=true). 
2. Hyperlink the `First Name Last Name` fields with the contributors' LinkedIn profile.
3. Hyperlink the `email` and `Github` fields with the information provided.

### PR DESCRIPTION
**Pull request by sasha-go**
## Description
This PR hyperlinks the maintainers' and contributors' contact information into Teams.md.

image

## Related Issues
Closes #15 
Relates to #12

### CODE REVIEW

**Comment by sasha-go** [comment]
@CBID2 I copied in your markdown for now while your PR is being reviewed. Let me know your thoughts on the formatting, can update to what makese the most sense.

File: Teams.md [hunk 1]
```diff
@@ -4,34 +4,34 @@
...
 
 ## Americas01
 
 ---
 
 ### Co-Team Leads(Maintainers)
 
-- First Name  Last Name  Preferred Name  GitHub username  email  location  email timezone
-- First Name  Last Name  Preferred Name  GitHub username  email  location  email timezone
+- [Chrissy Belzie](https://www.linkedin.com/in/christinebelzie/)  Preferred Name  [ChrissyCodes](https://github.com/CBID2)  [christinebelzie@gmail.com](mailto:christinebelzie@gmail.com)  location  timezone
```
Comment (CBID2) [inline Teams.md:27]:
```suggestion
- [Christine Belzie](https://www.linkedin.com/in/christinebelzie/)  Chrissy
[ChrissyCodes](https://github.com/CBID2)  [christinebelzie@gmail.com](mailto:christinebelzie@gmail.com)  NY  EST
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
It looks ok so far @sasha-go. I just gave feedback on how to improve the section where my contact information is

File: Teams.md (hunk 1 above)
Comment (sasha-go) [inline Teams.md]: Thanks! Updated now. I left preferred names and a couple of the timezones as the default for now they're currently unknown based off the spreadsheet information. 

image

**Comment by GingerKiwi** [comment]
Thanks everyone!

File: Teams.md (hunk 1 above)
Comment (GingerKiwi) [reply Teams.md]: Hyperlinks are an excellent idea!

**Pull request merged**